kmir/src/tests/integration/data/convert-body/panic/serialized.json

```diff
@@ -1 +1 @@
-{"body":[{"blocks":[{"statements":[],"terminator":{"kind":{"Call":{"func":{"Constant":{"span":0,"user_ty":null,"literal":{"kind":"ZeroSized","ty":{"id":0,"kind":{"RigidTy":{"FnDef":{"FnDef":[1,"std","std::rt::begin_panic",[{"Type":{"id":1,"kind":{"RigidTy":{"Ref":[{"kind":"ReErased"},{"id":4,"kind":{"RigidTy":"Str"}},"Not"]}}}}],{"value":{"inputs_and_output":[{"id":1,"kind":{"RigidTy":{"Ref":[{"kind":"ReErased"},{"id":4,"kind":{"RigidTy":"Str"}},"Not"]}}},{"id":3,"kind":{"RigidTy":"Never"}}],"c_variadic":false,"safety":"Safe","abi":"Rust"},"bound_vars":[]}]}}}},"id":0}}},"args":[{"Constant":{"span":1,"user_ty":null,"literal":{"kind":{"Allocated":{"bytes":[0,0,0,0,0,0,0,0,14,0,0,0,0,0,0,0],"provenance":{"ptrs":[[0,{"Memory":{"bytes":[101,120,112,108,105,99,105,116,32,112,97,110,105,99],"provenance":{"ptrs":[]},"align":1,"mutability":"Not"}}]]},"align":8,"mutability":"Mut"}},"ty":{"id":1,"kind":{"RigidTy":{"Ref":[{"kind":"ReErased"},{"id":4,"kind":{"RigidTy":"Str"}},"Not"]}}},"id":1}}}],"destination":{"local":1,"projection":[]},"target":null,"unwind":"Continue"}},"span":2}}],"locals":[{"ty":{"id":2,"kind":{"RigidTy":{"Tuple":[]}}},"span":3,"mutability":"Mut"},{"ty":{"id":3,"kind":{"RigidTy":"Never"}},"span":2,"mutability":"Mut"}],"arg_count":0,"var_debug_info":[],"spread_arg":null,"span":4},null]}
+{"gallocs":[{"Memory":{"bytes":[101,120,112,108,105,99,105,116,32,112,97,110,105,99],"provenance":{"ptrs":[]},"align":1,"mutability":"Not"}}],"body":[{"blocks":[{"statements":[],"terminator":{"kind":{"Call":{"func":{"Constant":{"span":0,"user_ty":null,"literal":{"kind":"ZeroSized","ty":{"id":0,"kind":{"RigidTy":{"FnDef":{"FnDef":[1,"std","std::rt::begin_panic",[{"Type":{"id":1,"kind":{"RigidTy":{"Ref":[{"kind":"ReErased"},{"id":4,"kind":{"RigidTy":"Str"}},"Not"]}}}}],{"value":{"inputs_and_output":[{"id":1,"kind":{"RigidTy":{"Ref":[{"kind":"ReErased"},{"id":4,"kind":{"RigidTy":"Str"}},"Not"]}}},{"id":3,"kind":{"RigidTy":"Never"}}],"c_variadic":false,"safety":"Safe","abi":"Rust"},"bound_vars":[]}]}}}},"id":0}}},"args":[{"Constant":{"span":1,"user_ty":null,"literal":{"kind":{"Allocated":{"bytes":[0,0,0,0,0,0,0,0,14,0,0,0,0,0,0,0],"provenance":{"ptrs":[[0,0]]},"align":8,"mutability":"Mut"}},"ty":{"id":1,"kind":{"RigidTy":{"Ref":[{"kind":"ReErased"},{"id":4,"kind":{"RigidTy":"Str"}},"Not"]}}},"id":1}}}],"destination":{"local":1,"projection":[]},"target":null,"unwind":"Continue"}},"span":2}}],"locals":[{"ty":{"id":2,"kind":{"RigidTy":{"Tuple":[]}}},"span":3,"mutability":"Mut"},{"ty":{"id":3,"kind":{"RigidTy":"Never"}},"span":2,"mutability":"Mut"}],"arg_count":0,"var_debug_info":[],"spread_arg":null,"span":4},null]}
```

Comment:
I think it might be the situation that the JSON is formatted slightly differently. The GlobalAllocs map will follow the bodies since it is populated by serializing the bodies (although that can be changed in if necessary). And the bodies would be within the context of the crate and function that is calling it. Not completely ironed out but something like 
```
{
    "name": crate_name,
    "items": [{
        "name": main,
        "id": 0,
        "kind": Fn,
        "body": [...],
    }]
},
{
    "gallocs": [...]
}
```
I'll work on getting this output working properly asap

Reply:
I already worked on this, and can submit a a separate PR that accepts a json of this format.